cli_reference/basic_cli_operations.adoc

```diff
@@ -754,3 +717,22 @@ link:https://access.redhat.com/errata/RHSA-2015:1650[For security purposes], the
 the command is executed by a `cluster-admin` user. Administrators can SSH into
 a node host, then use the `docker exec` command on the desired container.
 ====
+
+[[troubleshooting-cli]]
+
+== Troubleshooting oc
+At any point in time you can get more verbosed output from any command by increasing
+the loglevel using `-v=X` flag. By default, the loglevel is set to 0, but you can
+set its value from 0 to 10. Below you shall find explanation what you might expect
+at each loglevel.
```

Comment:
I would remove this last sentence and replace it with this just before the list:

.Overview of each loglevel

that should render as an example heading